users/forms.py

```diff
@@ -0,0 +1,10 @@
+
+from django.contrib.auth.forms import UserCreationForm
+from .models import CustomUser
+
+
+class CustomUserForm(UserCreationForm):
```

Comment:
Why you need this form definition?

Reply:
I have used this form in Custom User Admin Modal for custom fields while adding a new user via Django admin.

Reply:
You don't need this `CustomUserForm` since all you are doing is declaring fields of the model in it.